### PR TITLE
Adjustement of alert colors, fontsize and buttons contrasts

### LINKED ISSUE #101: Issues in develop branch
I found some issues while browsing the preview on the develop branch:

1. Low contrast link color in informational alert:

Screenshot 2020-08-03 at 17 07 59

2. Blue link in green alert, again low contrast (and arguably ugly :P)

Screenshot 2020-08-03 at 17 08 44

3. Low contrast in description text & icon color

Screenshot 2020-08-03 at 17 09 42

4. The warning text is low contrast; the warning color background looks a little "muddy" to me, perhaps we should pick a more orangey yellow instead.

Screenshot 2020-08-03 at 17 10 14

5. The warning badge looks even more muddy

Screenshot 2020-08-03 at 17 11 54

6. The warning buttons suffer from the same color/contrast problems:

Screenshot 2020-08-03 at 17 12 40

7. Again, low contrast:

Screenshot 2020-08-03 at 17 13 51

8. Rounded background in the selected pill looks weird:

Screenshot 2020-08-03 at 17 14 52

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | We wants to adjust some colors, fontsize and spacing
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes PrestaShop/PrestaShop#17177  / Fixes #101
| How to test?  | Check alert colors, font-sizes and spacing as in the issue, if you don't want to clone the repo you can see the uikit there : https://neomaking.github.io/prestashop-ui-kit/

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Sorry, I'm forced to wait for this PR as we noticed that @prestascott modified a lot of points afterwhile...

**Comment by PierreRambaud** [comment]
> Sorry, I'm forced to wait for this PR as we noticed that @prestascott modified a lot of points afterwhile...

If many things changed after it has been defined, it must be in a new pull request :thinking:

**Comment by NeOMakinG** [comment]
As you want, but it's aiming some things similiar than this one, this means that we need to merge this one with probably wrong things and correct these in another PR

**Comment by PierreRambaud** [comment]
> As you want, but it's aiming some things similiar than this one, this means that we need to merge this one with probably wrong things and correct these in another PR

Sad story :(

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

File: scss/_growl.scss [hunk 1]
```diff
@@ -35,18 +35,26 @@ $component-name: growl;
   }
 
   &.#{$component-name}-default {
-    background: $info;
+    color: $gray-700;
+    background: map-get($alerts-background-colors, success);
+    border: 1px solid map-get($alerts-second-colors, success);
```
Comment (Progi1984) [inline scss/_growl.scss:40]:
```suggestion
&.#{$component-name}{
  color: $gray-700;
  &-default {
    background: map-get($alerts-background-colors, success);
    border: 1px solid map-get($alerts-second-colors, success);
  }
```

And applied below

**Review by Progi1984 — COMMENTED** [review]
A small comment

File: scss/_growl.scss (hunk 1 above)
Comment (NeOMakinG) [reply scss/_growl.scss:40]: Nop, can't do this as the default jquery growl has a color property on every single classes, overwise I would need to use an !important which I don't want to

Comment (Progi1984) [reply scss/_growl.scss:40]: oh ok.

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @sarahdib

**Pull request merged**